domain/modelconfig/service/service.go

```diff
@@ -336,7 +403,7 @@ func (s *Service) updateModelConfigValidator(
 			validators.SpaceChecker(&spaceValidator{
 				st: s.st,
 			}),
-			validators.SecretBackendChecker(&dummySecretsBackendProvider{}),
+			validators.SecretBackendChecker(modelType),
```

Comment:
I can't see this change in the PR? Is there a file missing?

Reply:
This is in the existing validators.go along with all the others.